build.sbt

```diff
@@ -29,8 +29,9 @@ libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-actor" % "2.4.1",
   "com.google.guava" % "guava" % "19.0",
   "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
-  "com.fasterxml.jackson.core" % "jackson-core" % "2.7.0",
```

Comment:
In Logistics, we still need Jackson ObjectMapper. The reason is that the majority of our projects is written in Java with Spring Boot. We could think about introducing an interface for deserialization purposes. However, it is a strong requirement from our site that we still have a chance to use it.